tiledb/sm/config/config.h

```diff
@@ -717,6 +727,9 @@ class Config {
   /** Stores the parameters set by the user. */
   std::set<std::string> set_params_;
 
+  /** Temporary storage for lookup of a config value set by the environment. */
```

Comment:
Hmmm this isn't thread safe. I think we need to go a whole different way here...